lib/pure/asynchttpserver.nim

```diff
@@ -243,10 +247,14 @@ proc processRequest(
       if contentLength > server.maxBody:
         await request.respondError(Http413)
         return false
-      request.body = await client.recv(contentLength)
-      if request.body.len != contentLength:
-        await request.respond(Http400, "Bad Request. Content-Length does not match actual.")
-        return true
+
+      request.contentLength = contentLength
+      if server.handleBody == true: # if false leave the body for the developer to do whatever he wants with it.
```

Comment:
``server.handleBody`` is already of type ``bool`` so the condition should be ``if server.handleBody``.

Reply:
Ok, you're right